stl/inc/ranges

```diff
@@ -776,6 +777,72 @@ namespace ranges {
         };
     };
 
+    template <class _Fn, class... _Types>
+    class _Range_closure : public _Pipe::_Base<_Range_closure<_Fn, _Types...>> {
+    public:
+        // We assume that _Fn is the type of a customization-point object. That means

```

Comment:
Hyper nitpick: "customization-point" isn't hyphenated in the Standard or elsewhere in the STL.